content/index.hbs

```diff
@@ -31,7 +31,8 @@
 				<div class="column is-three-fifths">
 					<blockquote>
 						<p>Welcome to the home of Air Force ROTC's Detachment 165, based out of the Georgia Institute of Technology and host to 11 cross-town schools in the metro Atlanta area. Since its inception in the 1920s, Detachment 165 has commissioned over 1,700 officers for the United States Air Force and was recently ranked in the top 10% of AFROTC detachments nationwide. The key to our success is simple, and can be identified easily as the three core values of the Air Force: <em>Integrity First</em>, <em>Service Before Self</em>, and <em>Excellence in All We Do</em>. Detachment 165's mission is to train cadets and develop leaders who will excel in the U.S. Air Force and U.S. Space Force and go on to positively impact their units, their airmen, and the world.</p>
-						<cite>&mdash; C/Col Clair Hill, 165th Cadet Wing Commander, Spring 2020</cite>
+						<p>Spring 2021 Vision Statement: Foster leadership development, personal growth, and esprit de corps for cadets at all levels, through proactive planning, rigorously documented and upheld standards, and follow-through on innovative approaches to required objectives. Preparing cadets for Field Training, next level of leadership in the wing.</p>

```

Comment:
To be blunt, does anyone really care what our internal vision statement is? It also really doesn't fit into the tone of the previous paragraph and makes the whole thing very long.